src/adapters/arcgis-maps-sdk.adapter.ts

```diff
@@ -0,0 +1,247 @@
+	 */
+	public setDoubleClickToZoom(enabled: boolean) {
+		this._zoomEnabled = enabled;
+	}
+
+	/**
+	 * Renders GeoJSON features on the map using the provided styling configuration.
+	 * @param changes An object containing arrays of created, updated, and unchanged features to render.
+	 * @param styling An object mapping draw modes to feature styling functions
+	 */
+	public render(changes: TerraDrawChanges, styling: TerraDrawStylingFunction) {
+		changes.created.forEach((createdFeature) => {
+			this.addFeature(createdFeature, styling);
+		});
+
+		changes.updated.forEach((updatedFeature) => {
+			this.removeFeatureById(updatedFeature.id);
+			this.addFeature(updatedFeature, styling);
+		});
+
+		changes.deletedIds.forEach((deletedId) => {
+			this.removeFeatureById(deletedId);
+		});
+	}
+
+	/**
+	 * Clears the map and store of all rendered data layers
+	 * @returns void
+	 * */
+	public clear() {
+		this._featureLayer.graphics.removeAll();
+	}
+
+	private initializeMapViewListeners() {
+		this._mapView.on("drag", (event) => {
+			if (!this._dragEnabled) {
+				event.stopPropagation();
+			}
+		});
+		this._mapView.on("double-click", (event) => {
+			if (!this._zoomEnabled) {
+				event.stopPropagation();
+			}
+		});
+	}
+
+	private removeFeatureById(id: string | number | undefined) {
+		const feature = this._featureLayer.graphics.find(
+			(g) => g.attributes[this._featureIdAttributeName] === id
+		);
+		this._featureLayer.remove(feature);
+	}
+
+	private addFeature(
+		feature: GeoJSONStoreFeatures,
+		styling: TerraDrawStylingFunction
+	) {
+		const { coordinates, type } = feature.geometry;
+		const style = styling[feature.properties.mode as string](feature);
+
+		let symbol: Symbol | undefined = undefined; // eslint-disable-line @typescript-eslint/ban-types
+		let geometry: Geometry | undefined = undefined;
+
+		switch (type) {
+			case "Point":
+				geometry = new this._lib.Point({
+					latitude: coordinates[1],
+					longitude: coordinates[0],
+				});
+				symbol = new this._lib.SimpleMarkerSymbol({
+					color: this.getColorFromHex(style.pointColor),
+					size: style.pointWidth,
+					outline: {
+						color: this.getColorFromHex(style.pointOutlineColor),
+						width: style.pointOutlineWidth,
+					},
+				});
+				break;
+			case "LineString":
+				geometry = new this._lib.Polyline({ paths: [coordinates] });
+				symbol = new this._lib.SimpleLineSymbol({
+					color: this.getColorFromHex(style.lineStringColor),
+					width: style.lineStringWidth,
+				});
+				break;
+			case "Polygon":
+				geometry = new this._lib.Polygon({ rings: coordinates });
+				symbol = new this._lib.SimpleFillSymbol({
+					color: this.getColorFromHex(
+						style.polygonFillColor,
+						style.polygonFillOpacity
+					),
+					outline: {
+						color: this.getColorFromHex(style.polygonOutlineColor),
+						width: style.polygonOutlineWidth,
+					},
+				});
+				break;
+		}
```

Comment:
The following changes will bring the styling in line with the other adapters :) 

```typescript
		switch (type) {
			case "Point":
				geometry = new this._lib.Point({
					latitude: coordinates[1],
					longitude: coordinates[0],
				});
				symbol = new this._lib.SimpleMarkerSymbol({
					color: this.getColorFromHex(style.pointColor),
					size: style.pointWidth * 2 + "px",
					outline: {
						color: this.getColorFromHex(style.pointOutlineColor),
						width: style.pointOutlineWidth + "px",
					},
				});
				break;
			case "LineString":
				geometry = new this._lib.Polyline({ paths: [coordinates] });
				symbol = new this._lib.SimpleLineSymbol({
					color: this.getColorFromHex(style.lineStringColor),
					width: style.lineStringWidth + "px",
				});
				break;
			case "Polygon":
				geometry = new this._lib.Polygon({ rings: coordinates });
				symbol = new this._lib.SimpleFillSymbol({
					color: this.getColorFromHex(
						style.polygonFillColor,
						style.polygonFillOpacity
					),
					outline: {
						color: this.getColorFromHex(style.polygonOutlineColor),
						width: style.polygonOutlineWidth + "px",
					},
				});
				break;
```

Reply:
Great, I've added that.